configs/bonitasoft.json

```
@@ -1,12 +1,45 @@
 {
   "index_name": "bonitasoft",
   "start_urls": [
-    "https://bonitasoft.github.io"
+    {
+      "url": "https://bonitasoft.github.io/bonita/7.8/",
+      "tags": ["7.8", "bonita"]
+    },
+    {
+      "url": "https://bonitasoft.github.io/bonita/7.9/",
+      "tags": ["7.9", "bonita"]
+    },
+    {
+      "url": "https://bonitasoft.github.io/bonita/7.10/",
+      "tags": ["7.10", "bonita"]
+    },
+    {
+      "url": "https://bonitasoft.github.io/bonita/7.11/",
+      "tags": ["7.11", "bonita"]
+    },
+    {
+      "url": "https://bonitasoft.github.io/bonita/7.12/",
+      "tags": ["7.12", "bonita"]
+    },
```

```suggestion
    {
      "url": "https://bonitasoft.github.io/bonita/(?P<version>.*?)/",
      "tags": ["bonita"],
      "variables": {
        "version": ["7.8", "7.9", "7.10", "7.11", "7.12"]
      }
    },
```

This is how we usually do, which allow you to filter your search by `version`